docs/versioned-plugins/outputs/elasticsearch-index.asciidoc

```diff
@@ -104,6 +104,25 @@ include::{include_path}/version-list-intro.asciidoc[]
 | <<v9.1.3-plugins-outputs-elasticsearch,v9.1.3>> | 2018-05-15
 | <<v9.1.2-plugins-outputs-elasticsearch,v9.1.2>> | 2018-05-01
 | <<v9.1.1-plugins-outputs-elasticsearch,v9.1.1>> | 2018-04-06
+| <<v9.0.3-plugins-outputs-elasticsearch,v9.0.3>> | 2018-02-08
```

Comment:
v7.3.2 through v9.0.3 were truncated at some point, and are now being adding back in. Follow-up on this behavior.